src/appleseed/foundation/math/scalar.h

```diff
@@ -71,8 +71,9 @@ static const double TwoPi           =  6.2831853071795865;      // 2 * Pi
 static const double HalfPi          =  1.5707963267948966;      // Pi / 2
 static const double RcpPi           =  0.3183098861837907;      // 1 / Pi
 static const double RcpTwoPi        =  0.1591549430918953;      // 1 / (2 * Pi)
+static const double RcpFourPi       =  0.0795774715459477;      // 1 / (4 * Pi)
 static const double RcpHalfPi       =  0.6366197723675813;      // 1 / (Pi/2) = 2 / Pi
-static const double RcpPiSq         =  0.1013211836423377;      // 1 / (Pi^2) = (1 / Pi)^2
+static const double RcpPiSq         =  0.1013211836423378;      // 1 / (Pi^2) = (1 / Pi)^2
```

Comment:
Woops, sorry, not properly awaken: the expressions are already evaluated :) So no, `static const` and `#define` are essentially the same. It's simply a question of preference.